src/Lightbox.js

```diff
@@ -241,13 +241,13 @@ class Lightbox extends Component {
 				*/}
 				<img
 					className={css(this.classes.image)}
-					onClick={!!onClickImage && onClickImage}
+					onClick={onClickImage}
 					sizes={sizes}
 					alt={image.alt}
 					src={image.src}
 					srcSet={srcset}
 					style={{
-						cursor: this.props.onClickImage ? 'pointer' : 'auto',
+						cursor: onClickImage ? 'pointer' : 'auto',
```

Comment:
No need for this.props since this has been done earlier in `const {...} = this.props;`